Library/Homebrew/requirements/cctools_requirement.rb

```diff
@@ -0,0 +1,8 @@
+class CctoolsRequirement < Requirement
```

Comment:
@mikemcquaid We reverted to the original system of hooking in `cctools`, so this is still relevant.